src/api/app/controllers/webui/user_controller.rb

```diff
@@ -14,6 +14,8 @@ def index
 
   def logout
     logger.info "Logging out: #{session[:login]}"
+    key = "ldap_cache_userpasswd:" + session[:login]
+    Rails.cache.delete(key)
```

Comment:
btw I couldn't find any ldap_cache_userpasswd string in our git sources. Are you sure this cache key exists?